app/scripts/modules/amazon/serverGroup/configure/serverGroupCommandBuilder.spec.js

```diff
@@ -26,6 +27,7 @@ describe('awsServerGroupCommandBuilder', function() {
 
     it('initializes to default values, setting usePreferredZone flag to true', function () {
       var command = null;
+      this.settings.providers.aws.defaults.iamRole = '{{application}}IAMRole';
```

Comment:
My initial reaction is not to use the double curly brace '{{..}}' for a string substitution token since it already has a defined meaning within a Angular 1.x application and can lead to confusion.  With that said, if we have this as an already established pattern in the app then I'm okay with it.